book/src/external-provisioner.md

```diff
@@ -8,7 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min K8s Version | Max K8s Version | Recommended K8s Version
 --|--|--|--|--|--|--|--
-[external-provisioner v1.3.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.3.0) | [release-1.3](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.3.0 | v1.13 | - | v1.15
+[external-provisioner v1.3.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.3.1) | [release-1.3](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.3.1 | v1.13 | - | v1.15
```

Comment:
Sorry! I missed that there is already a 1.4.0 because it's not on the top of the "releases" page:
https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.4.0

Reply:
We also just released a bunch of patch versions for provisioner.

Reply:
fixed